libnmstate/sysctl.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import errno
+
+
+def enable_ipv6(dev):
+    disable_ipv6(dev, disable=False)
+
+
+def disable_ipv6(dev, disable=True):
```

Comment:
Here is a boolean argument that you do not like. Since the method is called disable and has a disable parameter, it should maybe better be called `configure_ipv6()` or `set_ipv6()`. Or make the argument the number and then have two functions for enabling/disabling that call this function with the right number.

Reply:
Sure.
I just copied this from vdsm.